### PR TITLE
[issue-426] add Security profile

### PR DESCRIPTION
**Pull request by armintaenzertng**
part of #426

### CODE REVIEW

**Review by meretp — CHANGES_REQUESTED** [review]
Thanks! I spotted some inconsistencies that need to be updated.

File: src/spdx_tools/spdx3/model/security/vuln_assessment_relationship.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from datetime import datetime
+from typing import Optional
+
+from spdx_tools.common.typing.dataclass_with_properties import dataclass_with_properties
+from spdx_tools.spdx3.model import Agent, Element, Relationship
+
+
+@dataclass_with_properties
+class VulnAssessmentRelationship(Relationship, ABC):
+    assessed_element: Optional[Element] = None
```
[the remark this answers is not in the document]
Comment (armintaenzertng) [reply src/spdx_tools/spdx3/model/security/vuln_assessment_relationship.py]: done

File: src/spdx_tools/spdx3/model/security/cvss_v2_vuln_assessment_relationship.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    ExternalIdentifier,
+    ExternalReference,
+    IntegrityMethod,
+)
+from spdx_tools.spdx3.model.security.vuln_assessment_relationship import VulnAssessmentRelationship
+
+
+@dataclass_with_properties
+class CvssV2VulnAssessmentRelationship(VulnAssessmentRelationship):
+    score: float
```
[the remark this answers is not in the document]
Comment (armintaenzertng) [reply src/spdx_tools/spdx3/model/security/cvss_v2_vuln_assessment_relationship.py]: done

File: src/spdx_tools/spdx3/model/security/vex_affected_vuln_assessment_relationship.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    ExternalReference,
+    IntegrityMethod,
+)
+from spdx_tools.spdx3.model.security.vex_vuln_assessment_relationship import VexVulnAssessmentRelationship
+
+
+@dataclass_with_properties
+class VexAffectedVulnAssessmentRelationship(VexVulnAssessmentRelationship):
+    action_statement: Optional[str] = None
+    action_statement_time: List[str] = field(default_factory=list)
```
[the remark this answers is not in the document]
Comment (armintaenzertng) [reply src/spdx_tools/spdx3/model/security/vex_affected_vuln_assessment_relationship.py]: done

File: src/spdx_tools/spdx3/model/security/vex_vuln_assessment_relationship.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+# SPDX-License-Identifier: Apache-2.0
+from abc import ABC, abstractmethod
+from typing import Optional
+
+from spdx_tools.common.typing.dataclass_with_properties import dataclass_with_properties
+from spdx_tools.spdx3.model.security.vuln_assessment_relationship import VulnAssessmentRelationship
+
+
+@dataclass_with_properties
+class VexVulnAssessmentRelationship(VulnAssessmentRelationship, ABC):
```
Comment (meretp) [inline src/spdx_tools/spdx3/model/security/vex_vuln_assessment_relationship.py:12]:
```suggestion
class VexVulnAssessmentRelationship(VulnAssessmentRelationship):
```

File: src/spdx_tools/spdx3/model/security/vuln_assessment_relationship.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import Optional
+
+from spdx_tools.common.typing.dataclass_with_properties import dataclass_with_properties
+from spdx_tools.spdx3.model import Agent, Relationship
+
+
+@dataclass_with_properties
+class VulnAssessmentRelationship(Relationship, ABC):
```
Comment (meretp) [inline src/spdx_tools/spdx3/model/security/vuln_assessment_relationship.py:13]:
```suggestion
class VulnAssessmentRelationship(Relationship):
```

**Review by meretp — COMMENTED** [review]
I spotted two more little things.

**Review by meretp — APPROVED** [review]
Thanks!

**Review by meretp — APPROVED** [review]
Thanks!

**Pull request merged**